mne/viz/ica.py

```diff
@@ -207,26 +209,61 @@ def _plot_ica_sources_evoked(evoked, picks, exclude, title, show):
     if title is None:
         title = 'Reconstructed latent sources, time-locked'
 
-    fig = plt.figure()
+    fig, axes = plt.subplots(1)
+    ax = axes
+    if isinstance(axes, plt.Axes):
```

Comment:
at this point don't you know that `axes` is a `plt.Axes` instance, since `squeeze=True` by default for `plt.subplots`?